styles.css

```diff
@@ -3,4 +3,110 @@ body{
 }
 h1{
     color:#6b37c1;
+}
+#services{
+    box-sizing: border-box; 
+    /*min-height: 100vh;*/
+    /* max-width: 250px; */
+    padding: 20px;
+    background-color:#bae1ff;
+}
+.flip-container{
+    perspective: 1000px;
+    margin: 3%;
+}
+.front-card-creativity h2 info-title{
+    margin: 0%;
+    font-family: Arial, Helvetica, sans-serif;
+    text-align: center;
+}
+
+.front-card-creativity{
+   font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+   font-size: 2em;
+   padding: 10px 10px 0 10px;
+}
+.flip-container:hover .flip-motion{
+    transform: rotateY(180deg);
+}
+.flip-container, .font-card, .back-card-creativity{
+    height: 250px;
+    width: 250px;
+}
+.flip-motion{
+    transition: 0.6s;
+    transform-style: preserve-3d;
+    position: relative;
+}
+.front-card, .back-card-creativity{
+      background-color:#370140;
+      color:#bae1ff;
+      backface-visibility:hidden;
+      position:absolute;
+      top: 0; 
+      left: 0; 
+      margin: 15px;
+      width: 200px;
+      height: 200px;
+      border-radius: 5%;
+      box-shadow: 14px 16px 20px 0px #370140e0, 0 6px 20px 0 rgb(0 0 0 / 19%);
+}
+.front-card{
+    z-index: 2;
+}
+.back-card-creativity{
+    transform:rotateY(180deg);
+   /* padding: 5%; */
+}
+
+h2, h3 {
+  font-size: 0.8rem;
+  vertical-align: middle;
+}
+.detailed-info{
+    vertical-align: middle;
+    padding: 1rem;
+    font-size: 8px;
```

Comment:
I noticed that you prefer using pixels over other units, which might not be the best choice for accessibility. 
Because if someone increases the font size on their screen, the size of the rest of the elements will remain the same when you use pixels. So maybe I'd be best practice to use rem or em units in places where you need to set a fixed width or height, since these units are relative to the font size.

Reply:
That's a very good point, ill change to only using 'rem' or 'em' 🌟